rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Volumes in Rancher
+layout: rancher-default-v1.6
+version: v1.6
+lang: en
+---
+
+## Volumes
+---
+
+Persistent volumes are a very important part of a stateful application. Rancher automates the installation and management of Docker volume plugins and supports a few in-house plugins. Each plugin is typically run inside of a container on every host comprising an environment. In Cattle terms, the plugin is a [global service]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/scheduling/#global-service).
+
+### Terminology
+
+The terms `volume plugin` and `volume driver` are used interchangeably in documentation and reference the same thing. 
+
+The term `storage driver` pertains to how containers and images are stored and managed on your Docker hosts. Examples are: aufs, btrfs, zfs, devicemapper. These drivers are outside the scope of this document and are discussed [here](https://docs.docker.com/engine/userguide/storagedriver/imagesandcontainers/).
+
+### Volume Scope
+
+There are three different scopes for volumes created in Rancher.
+
+#### Environment
+
+An environment-scoped volume is accessible to any container within that environment, irrespective of service/stack ownership. Suppose a shared volume with name `foo` was created. The name of the volume manifested on each host within the environment would also be `foo`. 
+
+For services created with V1 compose or Rancher UI, volumes may only be of environment scope.
+
+#### Stack
+
+A stack-scoped volume is accessible to any container in any service within the same stack. Suppose a shared volume with name `foo` was created inside stack `bar`. The name of the volume manifested on each hosts within the environment would be `bar_foo_<hash>`, where `<hash>` is randomly generated for the stack and will practically not collide with other hashes.
```

Comment:
> "is accessible to any container in any service within the same stack

is not accurate. We don't place any auth restrictions on the volume, so any container could mount this volume.

I'd say something like:
> A stack-scoped volume is owned and managed by the stack that defined it. When the stack is removed, its associated stack-scoped volumes will be removed. 

... and then go on with your name translation bit.